domain/lib/src/model/quota/quota_id.dart

```diff
@@ -29,8 +29,19 @@
 //  3 and <http://www.linshare.org/licenses/LinShare-License_AfferoGPL-v3.pdf> for
 //  the Additional Terms applicable to LinShare software.
 
-class QuotaId {
+import 'package:equatable/equatable.dart';
+
+class QuotaId extends Equatable {
   final String uuid;
 
   QuotaId(this.uuid);
+
+  factory QuotaId.initial() {
```

Comment:
what for?

Reply:
because when we fetch all shared spaces we don't have quotaId, so I need to null it